plugin.rb

```diff
@@ -284,13 +284,13 @@ class ::Group
 
     if name == "nobody"
       next results
-        .joins("LEFT JOIN assignments a ON a.topic_id = topics.id")
+        .joins("LEFT JOIN assignments a ON a.topic_id = topics.id AND active IS TRUE")
```

Comment:
Trivial but in SQL you can do `AND active` without the `IS TRUE`. I would never not approve for this, and it's up to you to use it or not.